libs/community/langchain_community/tools/azure_ai_services/image_analysis.py

```diff
@@ -68,35 +81,35 @@ def validate_environment(cls, values: Dict) -> Any:
                 f"Initialization of Azure AI Vision Image Analysis client failed: {e}"
             )
 
-        values["visual_features"] = [
-            VisualFeatures.TAGS,
-            VisualFeatures.OBJECTS,
-            VisualFeatures.CAPTION,
-            VisualFeatures.READ,
+        visual_features: List[str] = values.get("visual_features", ["TAGS"])
+        values["formatted_features"] = [
+            VisualFeatures[feat.upper()] for feat in visual_features
```

Comment:
I think we should leave the current defaults, we don't want the results to change for existing users. If a user wants to supply their own visual_features (or omit some), could something like this work?

```python
if values.get("visual_features") is None:
    values["visual_features"] = [
        VisualFeatures.TAGS,
        VisualFeatures.OBJECTS,
        VisualFeatures.CAPTION,
        VisualFeatures.READ,
    ]


```

Reply:
Note that if the user selects a visual feature that the code in `_image_analysis` does not yet support or parse from the result object, we may need to add that functionality as well.